### ISSUE TITLE
notification emails not being sent

### ISSUE THREAD

**Issue by IanMiddlebrook** [BiologicalRecordsCentre/UKBMS-online#225]
Hi @BirenRathod @DavidRoy 

Several new recorders are reporting that they didn't receive an email, either when they registered with the website or when their account was approved. This seems to be intermittent/inconsistent. 
Some have reported seeing an error message on the screen after they submit their registration, stating that an email could not be sent.

I've noticed that I get notification emails for the majority of new users, but not all, so clearly that email is also inconsistent.

Likewise, when I unblock new user accounts, I sometimes get a message that an email could not be sent. I've found that I need to block their account and then unblock it again before I can be vaguely confident that an email has been sent.

This has led to several new users thinking that they have been rejected by the scheme/website. 

Is it possible to identify the problem?

Also, is it possible for the notification emails that I receive to include the following information:
First Name
Last Name
Email address
Recording Branch
Intended Transects

This would save me some time when processing each new user.

Many thanks, Ian

**Comment by BirenRathod** [comment]
@IanMiddlebrook 
I have updated the library to support email execution. This should resolve the problem hopefully.

Also, I have added above mentioned fields, so you will see details of new user in the notification emails.

**Comment by IanMiddlebrook** [comment]
Thanks @BirenRathod 
The emails still seem to be a bit sporadic. There were 7 new accounts registered over the weekend, but I only received a notification email for 4 of them. Nothing for users 29812, 29814 & 29816.

Thanks for adding the fields to the notification emails, although the Email address is not coming through:
eg.
_Account details of MikeLewis
First name : Mike
Last name:  Lewis
Email address :
Recording branch : Dorset
Intended transects : Hod Hill_

Best wishes, Ian

**Comment by BirenRathod** [comment]
@IanMiddlebrook I have implemented the email address field now, so should work. 
I checked those users and not found any obvious problem in their accounts. Not sure, why you didn't receive email?

**Comment by IanMiddlebrook** [comment]
Hi @BirenRathod 

Thanks for sorting out the email address field - that's coming through now.

As for the notification emails, I don't think there are any problems with these accounts, just a problem with the website sending out emails. It's a problem that needs sorting, because it's leaving new users thinking that their account has not registered.

It happened to me again this morning when I unblocked a new account - I got a message that the email could not be sent. 

Thanks, Ian
